src/components/ContactList.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import RandomButton from './RandomButton';
+
+class ContactList extends Component {
+
+  constructor(props){
+    super();
+    this.state = {
+      Contacts: props.contacts,
+      contactsShowed: props.contacts.slice(0,5)
+    }
+  }
+
+  addRandomActor = () => {
+    let contactsShowedCopy = this.state.contactsShowed;
+    let index = Math.floor(Math.random() * (this.state.Contacts.length - 4) + 4);
+    contactsShowedCopy.push(this.state.Contacts[index]);
+    this.state.Contacts.splice(index,1);
+    this.setState( { contactsShowed: contactsShowedCopy } );
+  }
+
+  render() {
+
+    return(
+      
+      <div className="main">
+        <h1>IronContacts</h1>
+        <RandomButton clickToAddRandom={ () => this.addRandomActor()} />
```

Comment:
You don't need to add this arrow function, just passing the function from the class as a parameter would be enough. `clickToAddRandom={this.addRandomActor}`